_pages/install/integrations/cdc-integration.md

```diff
@@ -0,0 +1,247 @@
+    ;;GigyaProfileLink;"Profile Details"
+
+
+    # CMS Navigation Nodes
+    UPDATE CMSNavigationNode;$contentCV[unique=true];uid[unique=true];title[lang=$lang]
+    ;;GigyaProfileNavNode;"Profile Details"
+
+
+    #### Content for Spartacus like SiteLink slots etc ####
+
+    INSERT_UPDATE CMSLinkComponent;$contentCV[unique=true];uid[unique=true];name;url;&linkRef;&componentRef;target(code)[default='sameWindow'];restrictions(uid,$contentCV)
+    ;;HelpLink;Help Link;/faq;HelpLink;HelpLink;
+    ;;ContactUsLink;Contact Us Link;/contact;ContactUsLink;ContactUsLink
+    ;;WishListLink;Wish List Link;/my-account/wishlist;WishListLink;WishListLink;;loggedInUser
+    ;;OrdersLink;Orders Link;/my-account/orders;OrdersLink;OrdersLink;;loggedInUser
+
+
+    INSERT_UPDATE ContentSlot;$contentCV[unique=true];uid[unique=true];name;active;cmsComponents(uid,$contentCV)
+    ;;SiteLinksSlot;Slot contains some links;true;OrdersLink,WishListLink,StoreFinderLink,ContactUsLink,HelpLink
+
+
+    INSERT_UPDATE ContentSlotName;name[unique=true];template(uid,$contentCV)[unique=true];validComponentTypes(code);compTypeGroup(code)
+    ;SiteLinks;GigyaLoginPageTemplate;CMSLinkComponent;;
+
+
+    INSERT_UPDATE ContentSlotForTemplate;$contentCV[unique=true];uid[unique=true];position[unique=true];pageTemplate(uid,$contentCV)[unique=true];contentSlot(uid,$contentCV)[unique=true];allowOverwrite
+    ;;SiteLinksSlot-GigyaLoginPage;SiteLinks;GigyaLoginPageTemplate;SiteLinksSlot;true
+
+
+
+    ###### SiteContext Slot and Components ######
+    INSERT_UPDATE CMSSiteContextComponent;$contentCV[unique=true];uid[unique=true];name;context(code);&componentRef
+    ;;LanguageComponent;Site Languages;LANGUAGE;LanguageComponent
+    ;;CurrencyComponent;Site Currencies;CURRENCY;CurrencyComponent
+
+    INSERT_UPDATE ContentSlot;$contentCV[unique=true];uid[unique=true];name;active;cmsComponents(uid,$contentCV)
+    ;;SiteContextSlot;Site Context Slot;true;LanguageComponent,CurrencyComponent
+
+    INSERT_UPDATE ContentSlotName;name[unique=true];template(uid,$contentCV)[unique=true];validComponentTypes(code);compTypeGroup(code)
+    ;SiteContext;GigyaLoginPageTemplate;CMSSiteContextComponent;;
+
+    INSERT_UPDATE ContentSlotForTemplate;$contentCV[unique=true];uid[unique=true];position[unique=true];pageTemplate(uid,$contentCV)[unique=true];contentSlot(uid,$contentCV)[unique=true];allowOverwrite
+    ;;SiteContextSlot-GigyaLoginPage;SiteContext;GigyaLoginPageTemplate;SiteContextSlot;true
+    ```
+
+### Configuring Spartacus for SAP Customer Data Cloud Integration
+
+Perform the following steps after you have set up your Spartacus Storefront. For more information, see [Building the Spartacus Storefront from Libraries](https://sap.github.io/spartacus-docs/building-the-spartacus-storefront-from-libraries/).
+
+1. Edit the environment.ts file to enable the gigya feature by setting the value of **gigya** to **true**.
```

Comment:
This is instructions are specific to our app. You should instead document case when you have all spartacus modules configured in app.module.ts. Then you have to import GigyaModule and add it there + configure.